### PR TITLE
Update error handling

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

**Comment by rtasalem** [comment]
> ## [Quality Gate Failed](https://sonarcloud.io/dashboard?id=ffc-mpdp-frontend&pullRequest=152) **Quality Gate failed**
> Failed conditions  [0.0% Coverage on New Code](https://sonarcloud.io/component_measures?id=ffc-mpdp-frontend&pullRequest=152&metric=new_coverage&view=list) (required ≥ 80%)
> 
> [See analysis details on SonarCloud](https://sonarcloud.io/dashboard?id=ffc-mpdp-frontend&pullRequest=152)

Noticed this for the code coverage too, not sure if you wanted to merge but let me know and can approve.

**Comment by johnwatson484** [comment]
> > ## [Quality Gate Failed](https://sonarcloud.io/dashboard?id=ffc-mpdp-frontend&pullRequest=152) **Quality Gate failed**
> > Failed conditions  [0.0% Coverage on New Code](https://sonarcloud.io/component_measures?id=ffc-mpdp-frontend&pullRequest=152&metric=new_coverage&view=list) (required ≥ 80%)
> > [See analysis details on SonarCloud](https://sonarcloud.io/dashboard?id=ffc-mpdp-frontend&pullRequest=152)
> 
> Noticed this for the code coverage too, not sure if you wanted to merge with this but let me know and can approve.

@rtasalem - primarily this is because I've added the App Insights module that was missing.  As this repo's tests are setup to run in TypeScript then it would be an effort to write an effective test against it.  Given there's an issue and we have high confidence this app insights integration works as it's replicated in every other service, I'm happy that coverage isn't as standard.

However, in general there are coverage issues in this repo, so will track that as technical debt.  Also as this at some point needs to be converted from TypeScript, feels like a lot of the issues can be resolved during refactoring.

**Review by rtasalem — APPROVED** [review]
All good :)

File: docker-compose.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ services:
     container_name: ffc-mpdp-frontend
     environment:
       APPINSIGHTS_CLOUDROLE: ffc-mpdp-frontend-local
-      APPINSIGHTS_INSTRUMENTATIONKEY: ${APPINSIGHTS_INSTRUMENTATIONKEY}
+      APPINSIGHTS_CONNECTIONSTRING: ${APPINSIGHTS_CONNECTIONSTRING}
```
[the remark this answers is not in the document]
Comment (johnwatson484) [reply docker-compose.yaml:12]: There's no difference in terms of what this allows us to do.  Microsoft have deprecated the use of Instrumentation Keys to connect to App Insights, so we need to switch to Connection Strings instead.

Nothing changes when we do it, but if we don't do it by 2025, connectivity will fail

**Pull request merged**